pyproject.toml

```diff
@@ -45,6 +45,12 @@ requires = [
 [project]
 name = "SciPy"
 license = {file = "LICENSE.txt"}
+description = """
+SciPy (pronounced "Sigh Pie") is an open-source software for mathematics,
+science, and engineering. It includes modules for statistics, optimization,
+integration, linear algebra, Fourier transforms, signal and image processing,
+ODE solvers, and more.
+"""
```

Comment:
This looks like a setuptools regression, so shouldn't be needed to change - but it's fine to change, given that the current version in `setup.py` which ends up on https://pypi.org/project/scipy/ is quite poor.

Reply:
Yeah ideally the readme should be there, but I don't know what we want to do in the future so that we have something compatible here.